### PR TITLE
improve error email styling

### PR DESCRIPTION
**Pull request by dannyroberts**
(It's not actually this small, I just zoomed out so it'd fit on my screen.)

screen shot 2015-07-01 at 11 05 43 am

### CODE REVIEW

**Comment by dannyroberts** [comment]
@czue @TylerSheffels

File: corehq/apps/hqadmin/templates/hqadmin/email/error_email.html [hunk 1]
```diff
@@ -19,31 +19,24 @@
...
 {% if forloop.counter0 == 0 %}
-<div>
-    <h3>{{ line }}</h3>
-</div>
+<h3>{{ line }}</h3>
 {% elif 'site-packages' in line %}
-<div>
-    <small>{{ line }}</small>
-</div>
+<pre style="color: #DDDDDD">{{ line }}</pre>
```
Comment (czue) [inline corehq/apps/hqadmin/templates/hqadmin/email/error_email.html]: assume css renders fine in gmail? I had it in my head that it might not

**Comment by czue** [comment]
:+1: to merge pending one quesiton

File: corehq/apps/hqadmin/templates/hqadmin/email/error_email.html (hunk 1 above)
Comment (dannyroberts) [inline corehq/apps/hqadmin/templates/hqadmin/email/error_email.html]: Yup it renders fine. The screenshot is from google inbox, and I just checked that it works in normal gmail as well.

**Pull request merged**